### DISCUSSION TITLE
Unable to open new PR

### DISCUSSION THREAD

**Discussion by yoavmmn** [community/community#56874] [Pull Requests]
### Select Topic Area

Bug

### Body

I am trying to open a new PR and keep getting this error message:

> Pull request creation failed. Validation failed: was submitted too quickly  

And that's it. It just keeps happening. I've waited several minutes between retries and still unsucessful.

**Comment by NachmanBerkowitz** [comment]
Any luck? I am having the same issue.

**Comment by ElvenMonky** [comment]
Same issue here

**Comment by thomasXschneider-kry** [comment]
same issue here. new attempt on new branch doesn't work either

**Comment by emilburganov** [comment]
https://stackoverflow.com/questions/73723587/github-api-create-pull-request-validation-failed

**Comment by willsmythe** [comment]
The error message indicates a rate limiting problem.

How were you trying to create a PR? From your browser? From `gh` or your own app? Something else?

**Reply by thomasXschneider-kry** [reply]
I created the pull request via the GitHub extension for VS-CODE.
I've hit the [create PR] button approximately 5 seconds after I published the branch / pushed to github.
The pop-up window in VS-code, that shows the button and subsequently the progress got stuck with a loading bar afterwards.
When checking github in browser, i saw a red banner reading "Pull request creation failed. Validation failed: was submitted too quickly".

I was then unable to create ANY PR.
i tried
- creating a new branch, making a new commit, trying to create a new PR via browser (same repo)
- logging in & out of the github account
- a different browser

none of which helped.

it eventually resolved itself after waiting for ~2h
